### PR TITLE
added some svgs in docs

### PR DESCRIPTION
**Pull request by iArchitSharma**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 292a0d2758440605d399d307440681d79bfe6942 at: https://654751f9f4597f248d4be664--meshery-docs-preview.netlify.app

File: docs/assets/img/architecture/meshery-catalog-mockup.svg
Comment (leecalcote) [inline docs/assets/img/architecture/meshery-catalog-mockup.svg:1]: Remove this.

Comment (iArchitSharma) [reply docs/assets/img/architecture/meshery-catalog-mockup.svg:1]: Done!

**Comment by l5io** [comment]
🚀 Preview for commit 82089f1699bff2774e57ea0b63807fce2e9c169f at: https://6547b952a334a96120c3c7d4--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
@iArchitSharma thanks for posting these. 👏

**Pull request merged**